src/observable/fromEvent.ts

```diff
@@ -1,40 +1,60 @@
 import {Observable} from '../Observable';
+import {isEventTarget} from '../util/isEventTarget';
+import {isHTMLCollection} from '../util/isHTMLCollection';
+import {isNodeList} from '../util/isNodeList';
 import {tryCatch} from '../util/tryCatch';
 import {errorObject} from '../util/errorObject';
 import {Subscription} from '../Subscription';
 import {Subscriber} from '../Subscriber';
 
+export type EventEmmitterLike1 = {
```

Comment:
> addListener/removeListener is generally associated with NodeJS 

: NodeJS from @david-driscoll 's comment. Just trying to find name gives bit more clarity - so any suggestions would be appreciated.